server/src/main/java/org/elasticsearch/rest/action/admin/cluster/RestReloadSecureSettingsAction.java

```diff
@@ -73,7 +73,7 @@ public RestChannelConsumer prepareRequest(RestRequest request, NodeClient client
             .setNodesIds(nodesIds);
         request.withContentOrSourceParamParserOrNull(parser -> {
             if (parser != null) {
-                final NodesReloadSecureSettingsRequest nodesRequest = nodesRequestBuilder.request();
+                final NodesReloadSecureSettingsRequest nodesRequest = PARSER.parse(parser, null);
```

Comment:
This is the result of a master merge gone wrong in the feature branch : https://github.com/elastic/elasticsearch/commit/4780880af67ac9c315cd6882f26454644ed47f36#diff-0187f4b109fbd7256c987cceae7691caL76

This is the actual fix and the only change necessary. If the buildSrc changes need further discussion, we could just merge the fix and the rest of the test changes when ready.